lib/testsuite/selector-playground/playground-listener.js

```diff
@@ -0,0 +1,60 @@
+const {WebSocketServer} = require('ws');
```

Comment:
the file needs to be renamed into something like `websocket-server`. The server should only listen on `127.0.0.1` to ensure that we only accept requests from localhost.

Reply:
done 👍